core/scheduler/src/main/resources/application.conf

```diff
@@ -64,6 +64,11 @@ whisk {
     grpc {
       tls = "false"
     }
+    scheduling {
+      stale-threshold = "100 milliseconds"
+      check-interval = "100 milliseconds"
+      drop-interval = "10 seconds"
```

Comment:
This is another config I forgot to mention we'll need to tune so good to see it configurable here. Does 10 seconds make sense as the default if the built in max action duration is 60 seconds? If the namespace or action is throttled and can't create more containers, then may be waiting up to 60 seconds to process new activations. On the flip side I think this is used to be able to stop the scheduler once all queues are drained before stopping so I also understand why you would want this to be on the lower side

Reply:
The drop interval is to periodically check whether there are any stale activations.
https://github.com/apache/openwhisk/blob/master/core/scheduler/src/main/scala/org/apache/openwhisk/core/scheduler/queue/MemoryQueue.scala#L865

The retention time is controlled by the queue retention time which is 60s by default.
https://github.com/apache/openwhisk/blob/master/core/scheduler/src/main/resources/application.conf#L73

Reply:
I see okay. Maybe the max retention ms default should be the same as the default completion ack timeout which I think is three minutes based on the timeout calculation in code

Reply:
ok, so do you want me to update the default retention time to 3 mins?

Reply:
Yea I think that would be best unless you see a reason why it should be 1 min since you're more familiar with the architecture